File: src/main/java/com/example/demo/entity/Article.java
package com.example.demo.entity;


import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;
import lombok.NonNull;

@Entity
@Table(name="Articles")
@Data
public class Article {
  
  @Id
  @GeneratedValue(strategy=GenerationType.AUTO)
  @Column
  private long id;

  @Column
  @NonNull
  private String title;
  
  @Column
  private String summary;
  
  @Column
  @NonNull
  private String content;
  
  public Article() {
    title = "";
    content = "";
  }
}

File: src/main/ui/article.js
const React = require('react');

export default class Article extends React.Component{
	render() {
		return (
			<tr>
				<td>{this.props.article.title}</td>
				<td>{this.props.article.summary}</td>
				<td>{this.props.article.content}</td>
			</tr>
		)
	}
}